AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/Connection/DataStoreConnectionScenario1Tests.swift

```diff
@@ -203,6 +238,20 @@ class DataStoreConnectionScenario1Tests: SyncEngineIntegrationTestBase {
             }
         }
         wait(for: [deleteProjectSuccessful], timeout: TestCommonConstants.networkTimeout)
+
+        // TODO: Delete Team should not be necessary, cascade delete should delete the team when deleting the project.
+        // Once cascade works for hasOne, the following code can be removed.
```

Comment:
does this mean that this test will fail once we've fixed cascade deletions or will turn into a no-op?

Reply:
the test should succeed since deleting a deleted item returns success, so it will turn into a no-op